bubbles/bubbles.py

```diff
@@ -140,12 +139,17 @@ def put_liaison_equal(self, r0, x, y0, y1, h):
         '''
 
         l = abs(y1-y0)
+        assert 0.5*h <= r0 , f"bad h parameter, should be less or equal to {2*r0}"
```

Comment:
"bad" parameter actually works -- gives a weird looking bond but I thought it might be not useless